DiseasesExpanded/Cures/MutatingAntiviralConfig.cs

```diff
@@ -40,22 +40,22 @@ public GameObject CreatePrefab()
         {
             DefineRecipe(BasicForagePlantConfig.ID, 2);
             DefineRecipe(ForestForagePlantConfig.ID, 0.25f);
-            if(DlcManager.IsExpansion1Active())
+            if (DlcManager.IsExpansion1Active())
                 DefineRecipe(SwampForagePlantConfig.ID, 2 / 3.0f);
 
             MedicineInfo medInfo = new MedicineInfo(ID, EffectID, MedicineInfo.MedicineType.CureSpecific, null, new string[] { MutatingSickness.ID });
 
-            GameObject looseEntity = EntityTemplates.CreateLooseEntity(ID, 
-                STRINGS.CURES.MUTATINGANTIVIRAL.NAME, 
-                STRINGS.CURES.MUTATINGANTIVIRAL.NAME, 
+            GameObject looseEntity = EntityTemplates.CreateLooseEntity(ID,
+                STRINGS.CURES.MUTATINGANTIVIRAL.NAME,
+                STRINGS.CURES.MUTATINGANTIVIRAL.DESC,
                 1f,
-                false, 
```

Comment:
why changed to true?

Reply:
Some of the mod pills are measured in units, some - in gramms. In-game pills are all measured in units.